firmware/integration/rusefi_config.txt

```diff
@@ -871,8 +871,8 @@ custom pin_mode_e 1 bits,    U08,   @OFFSET@, [0:6], @@pin_mode_e_enum@@
 	brain_pin_e LIS302DLCsPin;
 
 
-	int tpsAccelLength;+How many engine cycles to consider?;"cycles",        1,     0,    1, 200,     0
-	float tpsAccelEnrichmentThreshold;+Maximum delta in TPS percentage over the 'length'. Actual TPS change has to be above this value in order to TPS/TPS acceleration to kick in.;"roc",        1,     0,    0, 200,     3
+	int tpsAccelLength;+This is the number of engine cycles that the TPS position chance can occur over, a longer duration will make the enrichment more active but too long may affect steady state driving, a good default is 30-60 cycles. ;"cycles",        1,     0,    1, 200,     0
```

Comment:
"chance" or "change"?

Reply:
Looks like a spell checker misclick to be me, will correct.